stable_baselines3/common/torch_layers.py

```diff
@@ -93,7 +94,11 @@ def forward(self, observations: th.Tensor) -> th.Tensor:
 
 
 def create_mlp(
-    input_dim: int, output_dim: int, net_arch: List[int], activation_fn: Type[nn.Module] = nn.ReLU, squash_output: bool = False
+    input_dim: int,
```

Comment:
I can't comment on none modified code but above this line on line 69 is a comment about probably using an MlpPolicy when the `asset is_image_space(observation_space)` fails. Perhaps we should change the error to include MultiInputPolicy?